storm/x86/system_calls.rb

```diff
@@ -93,7 +93,7 @@ def create_wrapper_c(system_calls)
         file.puts %(
       // Restore the stack after the function call
       "addl   \$4 * #{num_parameters}, %esp\\n"\
-      )
+)
```

Comment:
This fix removes some extraneous trailing whitespace in the generated code that really doesn't add much value.